features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Listing/ApiThumb.js

```diff
@@ -172,11 +183,15 @@ class ApiThumb extends React.Component {
                             </Typography>
                         </div>
                     </div>
+                    <div className={classes.thumbInfo}>
+                        <Typography variant="subheading" gutterBottom align="left">
+                            <StarRatingBar rating={this.state.rating}/>
```

Comment:
Shall we use object destructuring to take this (`this.state.rating`) value to a constant first.

i:e

```javascript
const rating = {this.state};

.
.
// Later . . 
<StarRatingBar rating={rating}/>
```

Reply:
Resolved